BigQuery/src/QueryResults.php

```diff
@@ -146,6 +146,9 @@ public function __construct(
      *           **Defaults to** `10000` milliseconds (10 seconds).
      *     @type int $maxRetries The number of times to poll the Job status,
      *           until the job is complete. By default, will poll indefinitely.
+     *     @type bool $returnRawResults Returns the raw data types returned from
```

Comment:
A common way to execute queries is through [`BigQueryClient::runQuery`](https://github.com/googleapis/google-cloud-php/blob/6dcdb5d36319dbf90b245ad4b9376bf6e807345d/BigQuery/src/BigQueryClient.php#L324) which does a little magic to hide the interaction with the `QueryResults` class. Could we please make sure to also surface configuration option there?

Reply:
Absolutely!

Reply:
Hey Dave. I started looking into this and was hoping you could help me come with an example? In the snippet below I used the `runQuery` command, get a result set back, and then pass the `returnRawResults` option while retrieving rows.

```
$queryJobConfig = $bigQuery->query($query);
$queryResults = $bigQuery->runQuery($queryJobConfig);

$rows = $queryResults->rows(['returnRawResults' => true]);
foreach ($rows as $row) {
   print_r($row);
}
```

Does this look like what you had in mind?

Reply:
Close, I was thinking like this:

```php
$queryResults = $bigQuery->runQuery($queryJobConfig, [
    'returnRawResults' => true // make sure this is surfaced in the `runQuery` documentation
]);

foreach ($queryResults as $row) {
    // ....
}
```

Reply:
@dwsupplee - The option is being passed via the runQuery result now 👍